### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by zercomnexus** [hankmorgan/UnderworldExporter#23]
Instructions say to download a zip file... there is no such zip file with that name in your repository. im assuming you mean to download the whole repository. this should be made clear instead of searching through all your files to try to find something that doesn't exist within your files.

the instructions state to load an exe file (uw.exe) ... there is no such exe file anywhere in the extracted folder...

basically trying to follow your instructions at this point is not possible in order to make this work. i'd like to give it a try, but without any guidelines that match what actually show up... it doesn't seem to matter if i have origin or GOG version... i won't be able to load them into unity using your exporter, no matter how ingenious or well put together the code may be.

**Comment by hankmorgan** [comment]
UW.exe is part of the original game files for Underworld 1. I.E. what you have downloaded from gog or origin.  I've updated the notes

Basic steps for the gog version
Install from gog to whatever folder you want.
Open that folder and in there or it's subfolders you will see a file called game.gog
Extract the game.gog file using a file extraction tool (eg 7zip) to a different folder.
In that new folder you will see full UW1 and UW2 game folders containing uw.exe and uw2.exe. Take note of this path.

On windows download and unzip Windows.x64.v1_06c.zip to a folder of your choosing.
Open config.ini in that folder and at the top of the page you will see the path settings for the games.

Enter the path (incl. trailing slash) you extracted the original game files to.
Run uwe.exe in the same folder as config.ini to begin.

**Comment by zercomnexus** [comment]
The files within the gog version do not have any contents that end in .gog
![image](https://user-images.githubusercontent.com/36425907/36344381-60b4b1b8-13ce-11e8-8148-77457e6449e7.png)
![image](https://user-images.githubusercontent.com/36425907/36344383-67baedd8-13ce-11e8-9f95-aa8eec24f1f2.png)

where did you find the Windows.x64.v1_06c.zip??? i have no idea where or even what this file is...
there also doesn't appear to be a uwe.exe file anywhere in your contents either....???

sorry, but these instructions are just as incomplete as the ones on the github page for this exporter.

**Comment by hankmorgan** [comment]
You download windows.x64.v1_06c.zip from [](https://github.com/hankmorgan/UnderworldExporter/releases)

The screenshot you have up there is of the DOSBOX folder bundled with the original game. I'm guessing you need to go up one folder 

 Now you say this is the gog version but why does the path say steamapps? Is this a steam version of underworld? I'm not familiar with such a version of the game.

**Comment by zercomnexus** [comment]
its just where i placed the gog version... i've moved it
![image](https://user-images.githubusercontent.com/36425907/36344766-f7137248-13d3-11e8-8f4e-b295b0afa218.png)
Found the gog.game extracted it, moved it to another folder.
configured Path_UW1=D:\Games\Ultima Underworld\Gog_game\UW (i don't need to add the exe in here i believe correct?).

**Comment by zercomnexus** [comment]
![image](https://user-images.githubusercontent.com/36425907/36344785-3ce5c1fe-13d4-11e8-8c58-826fade15a57.png)
Looks like you wanted a copy of the uw exe inside the exporter folder? and that config file is the one i put the path into "Path_UW1=D:\Games\Ultima Underworld\Gog_game\UW"

when running exe, it won't do anything inside win10.

**Comment by hankmorgan** [comment]
Ok

I see now where you are going wrong. You've downloaded the github repository for the source code and you are trying to run the dos era executable in that folder. 

Lets go back to the start of the process.

1. Download download windows.x64.v1_06c.zip from  [https://github.com/hankmorgan/UnderworldExporter/releases](https://github.com/hankmorgan/UnderworldExporter/releases) This is all you need to download to get started (without music)
2. Extract the zip file
3. Open config.ini in the folder you have extracted to.
4 .Now I can see you've found and extracted the game.gog file now so we'll skip over that step. So what you just need to do is repeat what you did to the config.ini file where you filled in the path to the original game as follows. 
Path_UW1=D:\Games\Ultima Underworld\Gog_game\UW\
Note I've added a \ to the end of the path
5. You should be ready to go now. Just run UWE.exe in the extracted folder with the config.ini

**Comment by zercomnexus** [comment]
ahhhh, the googles only brought me here :P 
ill see what i can do to make it work, but the exe... just wont start.

**Comment by Zibit-Meowe** [comment]
For my Gog install, if I did not install the sound packages, I received no sound on 1.06c.  Not sure if it's the same on Steam version as I haven't tried it.

zercomnexus, if you also receive no sound, from the releases page, download either the "UW1_SB16.zip" or "UW1_PSX.zip" or the basic midi one, "UW1_MIDI.zip".

Extract the files to the root folder of your 1.06c install...  ie, my install is c:\UnderworldUnity106c\, the place I extracted the .zip on releases for the main game.

Also add that to the lower section of the "config.ini", which is in the same root directory.  For example, mine says "UW1_Soundbank=C:\UnderworldUnity106c\UW1_SB16\" because I use the soundblaster pack.

Make sure to add the trailing " \ " or you will receive a black box screen when trying to load the game from inside the application.

Hope this helps.

**Comment by TheWrightStuff1994** [comment]
Just want to know how to fix this?
![untitled](https://user-images.githubusercontent.com/7861649/37560929-ad71c480-2a0f-11e8-9e25-58dac5871603.png)
Did i do something wrong?

**Comment by hankmorgan** [comment]
Change your resolution settings. The original artwork is based on a 320x200 screen so as it is currently configured the resolution you pick in game should be in a 4;3 ratio ( I'm not in a position to check at the moment but I think this is correct)

**Comment by TheWrightStuff1994** [comment]
Well should I change my computer resolution or the game resolution (and if it is the game resolution then how to do I that with the game?)

**Comment by hankmorgan** [comment]
The game resolution. When your run the game it should be an option on the first window that appears.

**Comment by chrixxxx** [comment]
I have the same problem with the GUI: it's all messed up, so that I can't use inventary or choose dialogues...  neither read text in the bottom panel.. everything is corrupted and messed up. 

I can't choose any resolution on the first (starting) window (the UNITY standard panel) except for the 1920x1080 (16:9 aspect ratio)..  my native one. Can you implement a setting for the resolution in the text config file ? It's DOZEN of YEARS i yearn to play this wonderful and revolutionary game again without dosbox and not to be limited to that crappy 320x200 resolution. Your port is what I awaited for... it's amazing.. but I can't play it!!

**Comment by hankmorgan** [comment]
I've had issues with any resolution that is not at a multiple of the original due to using original art files.

It's a thing I would like to fix given the time.

**Comment by PlanetStargazer** [comment]
I second chrixxxx's suggestion. Being able to set a resolution in the config file would be best. Or you should look into why some people can only pick non-4:3 formats when starting the game. I have the same issue. It only let's me pick 1280x800, 1440x900, 1600x1024 and 1680x1050. All of them look like in TheWrightStuff1994's screenshot.

**Comment by PlanetStargazer** [comment]
It would also be quite helpful if the mouselook would be temporarily disabled as soon as you pick something up. When it's dropped into the inventory it should then be re-enabled.

**Comment by chrixxxx** [comment]
> It would also be quite helpful if the mouselook would be temporarily disabled as soon as you pick something up. When it's dropped into the inventory it should then be re-enabled.

Yes, I totally agree. It's (needlessly) complicated to have to manually switch with the key shortcut everytime you pick something (there's a lot of stuff to pick in these games!), then switch back while exploring. And quite annoying too.

**Comment by hankmorgan** [comment]
I'll put this (mouselook) in as an option for the next release.

**Comment by hankmorgan** [comment]
This feature is now in the latest release.
